cmd/control/control.go

```diff
@@ -332,13 +389,33 @@ func startFunc(*cobra.Command, []string) error {
 		client.WithRootDataDir(rootDataDir),
 		client.WithReassignPortsIfUsed(reassignPortsIfUsed),
 		client.WithDynamicPorts(dynamicPorts),
-		client.WithNetworkID(networkID),
 	}
 
+	if upgradePath != "" {
```

Comment:
ok, probably you can include this in the other function

Reply:
wdym?